lib/matplotlib/backends/backend_pgf.py

```diff
@@ -316,8 +317,14 @@ def __init__(self):
                                      stdin=subprocess.PIPE,
                                      stdout=subprocess.PIPE,
                                      cwd=self.tmpdir)
-        except OSError:
-            raise RuntimeError("Error starting process '%s'" % self.texcommand)
+        except OSError as e:
+            if e.errno == errno.ENOENT:
+                raise OSError(errno.ENOENT, "Latex command not found. "
```

Comment:
I guess the idea was to discriminate between the "command not found" and "command returned with error" case, so errors when running the latex process are propagated as RuntimeError, not finding the latex command is an OS or environment error. That distinction is a new "feature" if you would like to say so. I don't really know if this is to be considered as breaking the API, since the behaviour on error is neither defined nor documented anywhere.